From a3339604bafa99f66e12687efc2c50d2e2df1276 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sun, 12 Dec 2021 20:03:04 +0100
Subject: [PATCH] compat/gettcpinfo.c: fix build

Fix the following build failure raised since commit
c8f861a0a07b0606a581f2658c71959ad9417923:

gettcpinfo.c: In function 'gettcpinfo':
gettcpinfo.c:101:42: error: expected declaration or statement at end of input
  101 | inline void gettcpinfo (int sock, struct ReportStruct *sample) {
      |                                          ^~~~~~~~~~~~

Fixes:
 - http://autobuild.buildroot.org/results/073e070478f0cda34784af96bb3a802c3aac8f72

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 compat/gettcpinfo.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compat/gettcpinfo.c b/compat/gettcpinfo.c
index 4e11e2a..1d3791a 100644
--- a/compat/gettcpinfo.c
+++ b/compat/gettcpinfo.c
@@ -99,4 +99,5 @@ inline void gettcpinfo (SOCKET sock, struct ReportStruct *sample) {
 };
 #else
 inline void gettcpinfo (int sock, struct ReportStruct *sample) {
+};
 #endif
-- 
2.33.0

